modules/mono/editor/bindings_generator.cpp

```diff
@@ -523,7 +523,27 @@ void BindingsGenerator::_append_xml_method(StringBuilder &p_xml_output, const Ty
 				p_xml_output.append(p_target_itype->proxy_name);
 				p_xml_output.append(".");
 				p_xml_output.append(target_imethod->proxy_name);
-				p_xml_output.append("\"/>");
+				p_xml_output.append("(");
+				bool first_key = true;
+				for (const ArgumentInterface &iarg : target_imethod->arguments) {
+					const TypeInterface *arg_type = _get_type_or_null(iarg.type);
+
+					ERR_FAIL_NULL(arg_type); // Argument type not found
```

Comment:
Does this mean the documentation is left incomplete? We probably don't want to exit here.

Reply:
That's a fair point; I'll adopt the `ERRPRINT` used below & continue the loop